lib/services/game_service.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:niira/models/game.dart';
+
+class GameService {
+  Game currentGame;
+  GameService({this.currentGame});
+
+  void setCurrentGame(Game _selectedGame) => currentGame = _selectedGame;
+
+  Game get getCurrentGame => currentGame;
```

Comment:
getter method might be overdoing it, it feels safer to call that then gameService.currentGame.
Happy to remove if you think is unneccessary, or let me know if im overthinking adding 1 extra line haha

Reply:
definitely worth pondering IMO - luckily Effective Dart has pondered this and has a suggestion! :-) 
https://dart.dev/guides/language/effective-dart/usage#dont-wrap-a-field-in-a-getter-and-setter-unnecessarily